viewing_party/party.py

```diff
@@ -1,23 +1,145 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movie = {}
 
+    if title and genre and rating:
+        movie["title"] = title
+        movie["genre"] = genre
+        movie["rating"] = rating
+        return movie
+    else:
+        return None

```

Comment:
Nice work with this method! Couple of small callouts. 

You could convey intent a little more explicitly by pulling "return None" into it's own guard clause with:
``` 
if not title and not genre and not rating:
      return None

# then proceed to create your movie dict
``` 

if the method were written like that then another coder could quickly realize "oh title/genre/rating can't be Falsy".

You can also create your dictionary with literal syntax and return it without first creating an empty dict on line 4:
```
return {
        "title": title,
        "genre": genre,
        "rating": rating        
    }
```